Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -1315,6 +1315,16 @@ public extension ChatChannelController {
             }
         }
     }
+    
+    func currentCooldownTime() -> Int {
```

Comment:
Please provide documentation describing what this function does. Something like: 
`/// Returns the current cooldown time for the channel. Returns 0 in case there is no cooldown active.`

Reply:
Done